src/main/java/com/mindsnacks/zinc/classes/ZincBundleDownloader.java

```diff
@@ -14,14 +16,18 @@
 public class ZincBundleDownloader implements PriorityJobQueue.DataProcessor<ZincCloneBundleRequest, ZincBundle> {
     private final ZincJobFactory mJobFactory;
     private final ZincCatalogsCache mCatalogs;
+    private final ZincManifestsCache mManifests;
 
-    public ZincBundleDownloader(final ZincJobFactory jobFactory, final ZincCatalogsCache catalogs) {
+    public ZincBundleDownloader(final ZincJobFactory jobFactory,
+                                final ZincCatalogsCache catalogs,
+                                final ZincManifestsCache manifests) {
         mJobFactory = jobFactory;
         mCatalogs = catalogs;
+        mManifests = manifests;
     }
 
     @Override
     public Callable<ZincBundle> process(final ZincCloneBundleRequest request) {
-        return mJobFactory.cloneBundle(request, mCatalogs.getCatalog(request.getSourceURL()));
+        return mJobFactory.cloneBundle(request, mCatalogs.getCatalog(request.getSourceURL()), mManifests);
```

Comment:
I did it that way because we already had a `getManifest` method inside the class and this was the less-disruptive way to adapt the code we had without changing the flow